app/src/main/java/org/p2p/wallet/debugdrawer/ConfigurationModule.kt

```diff
@@ -8,17 +8,17 @@ import org.koin.core.component.KoinComponent
 import org.koin.core.component.inject
 import org.p2p.wallet.common.InAppFeatureFlags
 import org.p2p.wallet.databinding.ViewDebugDrawerConfigureEnvironmentBinding
-import org.p2p.wallet.infrastructure.network.environment.EnvironmentManager
+import org.p2p.wallet.infrastructure.network.environment.NetworkEnvironmentManager
 
 class ConfigurationModule : DebugModuleAdapter(), KoinComponent {
 
-    private val environmentManager: EnvironmentManager by inject()
+    private val environmentManager: NetworkEnvironmentManager by inject()
```

Comment:
renamed due to this entity being responsible for *Network* Environment exactly, but not some abstract Environment as a whole